src/test/blockencodings_tests.cpp

```diff
@@ -282,7 +286,7 @@ BOOST_AUTO_TEST_CASE(EmptyBlockRoundTripTest)
 
     // Test simple header round-trip with only coinbase
     {
-        CBlockHeaderAndShortTxIDs shortIDs{block};
+        CBlockHeaderAndShortTxIDs shortIDs{block, 1341};
```

Comment:
really not a fan of these magic numbers, would prefer `rand_ctx.rand64()`

Reply:
I think that should be OK, assuming it doesn't happen to hit a case where there's a short-ID collision

Reply:
Addressed in 8df9d16